### PR TITLE
P2367R0 Remove misuses of list-initialization from Clause 24

### LINKED ISSUE #4658: [2021-06 LWG Motion 21] P2367R0 Remove misuses of list-initialization from Clause 24
[P2367R0](https://wg21.link/p2367r0)

### PR DESCRIPTION
**Pull request by jensmaurer**
Also fixes LWG3524.

Fixes #4658
Fixes cplusplus/papers#1042

### CODE REVIEW

**Review by jwakely — APPROVED** [review]
Ignoring the overfull hbox, this correctly applies the changes from the motion.

File: source/ranges.tex [hunk 1]
```diff
@@ -6249,7 +6257,7 @@
 \pnum
 \effects
 Initializes \exposid{base_} with \tcode{views::all(std::forward<R>(r))}, and
-\exposid{pattern_} with \tcode{single_view\{std::move(e)\}}.
+\exposid{pattern_} with \tcode{views::\linebreak single(std::move(e))}.
```
Comment (tkoeppe) [inline source/ranges.tex:6260]: @timsong-cpp: is this what you had in mind?

Comment (timsong-cpp) [reply source/ranges.tex:6260]: Yes, thanks!

Comment (tkoeppe) [reply source/ranges.tex:6260]: Thanks for pointing it out!

**Pull request merged**